microsoft/testsuites/network/common.py

```diff
@@ -128,7 +128,7 @@ def sriov_vf_connection_test(
 
     dest_ssh.enable_public_key(source_ssh.generate_key_pairs())
     # generate 200Mb file
-    source_node.execute("dd if=/dev/urandom of=large_file bs=100 count=0 seek=2M")
+    source_node.execute("dd if=/dev/zero of=large_file bs=200M count=1")
```

Comment:
I think we can continue to use urandom and reduce the block size & increase count